ocs_ci/ocs/resources/pod.py

```diff
@@ -198,20 +184,15 @@ def exec_ceph_cmd(self, ceph_cmd, format='json-pretty'):
             return [item for item in out if item]
         return out
 
-    def get_storage_path(self, storage_type='fs'):
+    def get_mount_path(self):
```

Comment:
why have you removed the option to return the block device path? This is being used in some tests

Reply:
I had tried this out and seem to have sent it without reverting to the original function. I'll fix this.